app/Dockerfile

```
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1.4
-FROM --platform=$BUILDPLATFORM python:3.10-alpine
+FROM --platform=$BUILDPLATFORM python:3.14.0a6-alpine3.21
```

Python 3.14 isn't officially supported yet, 3.13.2-alpine3.21 is latest supported version and Docker Scout reports no package vulns.
```suggestion
FROM --platform=$BUILDPLATFORM python:3.13.2-alpine3.21
```